helix-term/src/commands.rs

```diff
@@ -1301,14 +1302,19 @@ fn global_search(cx: &mut Context) {
                     editor.set_status("No matches found".to_string());
                     return;
                 }
+
                 let picker = FilePicker::new(
                     all_matches,
                     move |(_line_num, path)| {
-                        path.strip_prefix(&root)
-                            .unwrap_or(path)
+                        let relative_path = helix_core::path::get_relative_path(path)
                             .to_str()
                             .unwrap()
-                            .into()
+                            .to_owned();
+                        if current_path.as_ref().map(|p| p == path).unwrap_or(false) {
+                            format!("{} (*)", relative_path).into()
```

Comment:
It seemed to take in quite a lot of characters, I wonder if doing this is good or not, or maybe just `*`?